src/shx.js

```diff
@@ -11,6 +11,37 @@ objAssign.polyfill(); // modifies the global object
 
 shell.help = help;
 
+const convertSedRegex = args => {
+  const newArgs = [];
+  let lookingForSubstString = true;
+  args.forEach((arg) => {
+    // A regex or replacement string can be any sequence of zero or more
+    // (a) non-slashes or (b) escaped chars.
+    const escapedChar = '\\\\.'; // This may match an escaped slash (i.e., "\/")
+    const nonSlash = '[^/]';
+    const nonSlashSequence = `(?:${escapedChar}|${nonSlash})*`;
+    const sedPattern = `^s/(${nonSlashSequence})/(${nonSlashSequence})/(g?)$`;
+    const match = arg.match(new RegExp(sedPattern));
+    if (match && lookingForSubstString) {
+      const regexString = match[1].replace(/\\\//g, '/');
+      const replacement = match[2].replace(/\\\//g, '/').replace(/\\./g, '.');
+      const regexFlags = match[3];
+      if (!regexString) {
```

Comment:
Perhaps `regexString === ''` would be clearer here.

Reply:
done